cmd/api-errors.go

```diff
@@ -163,6 +163,8 @@ const (
 	ErrOperationTimedOut
 	ErrPartsSizeUnequal
 	ErrInvalidRequest
+	ErrBackendDown
+	ErrATimeUnsupported
```

Comment:
Where is this Error being used? if we are validating if backend supports atime in startup we don't need this.